resources.html

```diff
@@ -25,70 +25,75 @@ <h1 style="color:#075988; text-align: left;"> <b>Makerspace Lab Resources</b> </
+                })
+                .then(data => {
+                    let resources = data.resources;
+                    let resourceContainer = document.getElementById("resourceContainer");
+                    for (let i = 0; i < resources.length; i++) {
+                        resourceContainer.innerHTML += `
+                    <div class="container">
+                        <div class="flex-left">
+                            <h3>` + resources[i].title + `</h3>
+                            <img src="`+ resources[i].image + `" alt="picture">
+                        </div>
+                        <div class="flex-right">
+                            <p class="right-desc">`+ resources[i].description + `</p>
+                            <form action="https://forms.gle/YAFNM6VkJ1RztLpE6">
+                                <button class="resource-signup-btn">Sign up</button>
+                            </form>
+                            <form action="https://drive.google.com/file/d/1FRGu3IZK1Vzd5dPeTjfgYOC0jcVOMaGq/view?usp=sharing">
+                                <button class="resource-tutorial-btn"> Watch Tutorial</button>
+                            </form>
+                    </div>`
 
-                }
-            });
-    }
-</script>
-<div id="footer">
-    <div class="flex-footer">
-        <div id="flex-footer-el-1">
-            <div class="logo"> <img src="img\Duck.png" width="100" height="100" align="middle" /> <b>BEREA COLLEGE MAKERSPACE </b></div>
-            <div id="footer-links">
-                <div class="home"><a href= "index.html" style="text-align: right; color: white; padding: 100px">HOME</a> </div>
-                <div class="resources"> <a href="resources.html" style="text-align:right; color:white; padding:100px"> RESOURCES</a></div>
-            </div>
+                    }
+                });
+        }
+    </script>
+ 
 
-        </div>
-        <div id="flex-footer-el-2">
-            <div><b>CONTACT US</b></div>
-            <div class="phone">   +1 859-985-3000</div>
-            <div class="email"> bereacollegemakerspace@gmail.com </div>
-            <div class="college-name">Berea College</div>
-            <div class="address-line">Danforth Technology Building</div>
-            <div class="address-line">126 N. Main St</div>
-            <div class="address-line">Berea, KY 40404</div>
+    <div id="footer">
+        <div class="flex-footer">
+            <div id="flex-footer-el-1">
+                <div class="logo"> <img src="img\Duck.png" width="100" height="100" align="middle" /> <b>BEREA COLLEGE MAKERSPACE </b></div>
+                <div id="footer-links">
+                    <div class="home"><a href="index.html" style="text-align: right; color: white; padding: 100px">HOME</a> </div>
+                    <div class="resources"> <a href="resources.html" style="text-align:right; color:white; padding:100px"> RESOURCES</a></div>
+                </div>

```

Comment:
The 'id' and 'class' elements in these tags should be combine into one. i.e `<div id="footer"  #class="flex-footer")>`